config/cookstyle.yml

```
@@ -454,6 +454,15 @@ ChefCorrectness/InvalidPlatformInCase:
     - '**/metadata.rb'
     - '**/Berksfile'
 
+ChefCorrectness/InvalidDefaultAction:
+  Description: Default actions in resources should be symbols or an array of symbols.
```

Maybe this is better?

```suggestion
  Description: Default actions in resources should be a symbol or an array of symbols.
```